meanMedian/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"log"
+	"sort"
+	"sync"
+	"time"
+)
+
+func addNums(i int, w *sync.WaitGroup, ch chan int) {
+	time.Sleep(time.Second)
+	ch <- i
+	w.Done()
+}
+
+func meanMedianCal(ch chan int, ch2 chan bool, count int, sum int) {
+	var median = 0
+	temp := make([]int, 0)
+	for j := range ch {
+		temp = append(temp, j)
+		sort.Ints(temp)
+		count++
+		sum += j
+		log.Println(j)
+		mean := float64(sum) / float64(count)
+		time.Sleep(time.Second)
+		log.Printf("Mean=%.2f", mean)
+		if count%2 != 0 {
+			middle := (len(temp) - 1) / 2
+			median = temp[middle]
+			time.Sleep(time.Second)
+			log.Println("Median=", median)
+		} else {
+			middle := len(temp) / 2
+			median = (temp[middle] + temp[middle-1]) / 2
+			time.Sleep(time.Second)
+			log.Println("median=", median)
+
+		}
+
+	}
+	ch2 <- true
+
+}
+
```

Comment:
what will be the median for below input 

1, 3, 5, 7, 9, 2, 4, 6, 8, 15
as per me it should be 5.5 

can you check if this is the result.